packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -400,8 +400,11 @@ export class DocumentDeltaConnection
 	 * Disconnect from the websocket, and permanently disable this DocumentDeltaConnection and close the socket.
 	 * However the OdspDocumentDeltaConnection differ in dispose as in there we don't close the socket. There is no
 	 * multiplexing here, so we need to close the socket here.
+	 *
+	 * @param error - An optional error object. If provided, the connection will be closed with the specified error,
+	 * indicating an error-triggered disconnect. If not provided, the connection will be closed cleanly.
 	 */
-	public dispose() {
+	public dispose(error?: Error) {
```

Comment:
You can replace Error with IErrorBase which is fluid error interface.

Reply:
Ok. I didn't notice on interface we have this: dispose(error?: Error): void; So lets keep the type as Error.